lib/components/Atom/Calendar/__tests__/__snapshots__/date-picker.test.tsx.snap

```diff
@@ -150,7 +150,7 @@ exports[`DatePicker should render calendar with double navigation if next2Label
                   </abbr>
                 </div>
                 <div
-                  class="react-calendar__month-view__weekdays__weekday react-calendar__month-view__weekdays__weekday--current"
+                  class="react-calendar__month-view__weekdays__weekday"
```

Comment:
No idea why date-picker test case failed. No changed on the component, update the snapshot to pass the test